File: Lesson2/hw2/src/second/hw/Motorcycle.java
package second.hw;

public class Motorcycle extends Vehicle {
    private int numWheels;
    private int speed;

    public Motorcycle() {
        this.numWheels = 2;
        this.speed = 0;
    }

    public int getNumWheels() {
        return numWheels;
    }

    public int getSpeed() {
        return speed;
    }

    public void testDrive() {
        this.speed = 75;
    }

    public void park() {
        this.speed = 0;
    }
}

File: Lesson2/Motorcycle.py
from Lesson2.Vehicle import Vehicle


class Motorcycle(Vehicle):
    def __init__(self):
        self.num_wheels = 2
        self.speed = 0

    def get_num_wheels(self):
        return self.num_wheels

    def get_speed(self):
        return self.speed

    def test_drive(self):
        self.speed = 75

    def park(self):
        self.speed = 0
